machines/bigblade_pod_X1Y1_ruche_X16Y8_hbm/Makefile.machine.include

```diff
@@ -110,6 +110,10 @@ BSG_MACHINE_IO_EP_CREDITS             = 32
 BSG_MACHINE_IO_HOST_CREDITS           = $(BSG_MACHINE_IO_EP_CREDITS)
 BSG_MACHINE_IO_REMOTE_LOAD_CAP        = $(BSG_MACHINE_IO_EP_CREDITS)
 
+# Enable DRAM pod dimensions
+BSG_MACHINE_ENABLE_DRAM_PODS_X        = $(BSG_MACHINE_PODS_X)
+BSG_MACHINE_ENABLE_DRAM_PODS_Y        = $(BSG_MACHINE_PODS_Y)
```

Comment:
I think there should be a comment describing how these parameters work currently.

Reply:
Thanks for the suggestions @tommydcjung !

- I've added comments to all the machine files for the usage of this new flag
- I've updated the README.md under machines directory (similar to the manycore README.md) with more detailed information about these flags

Reply:
@tommydcjung Do you think we should change the flag name to `BSG_MACHINE_ENABLE_DRAM_PROGRAM_LOADING_PODS_X/Y`?